app/views/api/v1/models/_user.json.jbuilder

```diff
@@ -1,19 +1,20 @@
-json.id resource.id
-json.provider resource.provider
-json.uid resource.uid
-json.name resource.name
-json.display_name resource.display_name
+json.access_token resource.access_token.token
+json.account_id resource.active_account_user&.account_id
+json.availability_status resource.availability_status
 json.available_name resource.available_name
+json.avatar_url resource.avatar_url
+json.confirmed resource.confirmed?
+json.display_name resource.display_name
 json.email resource.email
-json.account_id resource.active_account_user&.account_id
+json.hmac_identifier resource.hmac_identifier
```

Comment:
Should we make this attribute optional? 
Since it's meant just for the chatwoot cloud ? we could add an 
`if GlobalConfig.get('CHATWOOT_INBOX_HMAC_KEY')['CHATWOOT_INBOX_HMAC_KEY']` ?

Reply:
Yeah, that makes sense, will update.

Reply:
Done.